app/src/main/cpp/src/components/transport_manager/src/bluetooth_le/ble_client.cc

```diff
@@ -0,0 +1,75 @@
+#include "transport_manager/bluetooth_le/ble_client.h"
+
+#include <thread>
+#include "utils/logger.h"
+
+SDL_CREATE_LOG_VARIABLE("Ble_Client")
+
+namespace{
+    const char* socket_name =  "./localBleReader";
+
+    char buffer[256];
+    bool thread_stop = false;
+
+    void writer_func(int socket_id)
+    {
+        while(!thread_stop)
+        {
+            sleep(5);
+            sprintf(buffer, "qwerty");
+            int n = write(socket_id, buffer, 8);
+            SDL_LOG_INFO("@@@BLE_Write " << n);
+        }
+    }
+}
+
+void BleClient::Init()
+{
+    SDL_LOG_TRACE("enter");
```

Comment:
@ychernysheva why not SDL_LOG_AUTO_TRACE ?

Reply:
Done in 1537bc4